acis_thermal_check/main.py

```diff
@@ -499,7 +503,8 @@ def write_states(self, outdir, states):
         states_table['pitch'].format = '%.2f'
         states_table['tstart'].format = '%.2f'
         states_table['tstop'].format = '%.2f'
-        states_table.write(outfile, format='ascii', delimiter='\t', overwrite=True)
+        states_table.write(outfile, format='ascii', delimiter='\t', overwrite=True,
```

Comment:
Also needs a comment. The fast writer fails because there is an object column (the transition keys) which makes it crash. The Python writer just takes the `str` of every entry and that gives the desired output.